scripts/verify_links.py

```
@@ -230,69 +230,100 @@ def get_max_workers():
         return default_workers
 
 
+def extract_links_from_json(json_file):
+    """Extracts all Hugging Face links from models_download.json."""
+    try:
+        with open(json_file, "r", encoding="utf-8") as f:
+            data = json.load(f)
+        
+        links = []
+        for category, urls in data.items():
```

Ruff (0.14.2)

Loop control variable `category` not used within loop body

Rename unused `category` to `_category`

(B007)